app.py

```diff
@@ -0,0 +1,44 @@
+import requests
+
+class PublicNeeds(object):
+    def __init__(self):
+        pass
+
+    def requesting(self, page=1, limit=1000, last_page=4):
+        '''
+        Kwarg page will be used in for to iterate from this number to last_page
+        Limit is also used with default value of 100 as recommended in api docs.
+        For now i willl use self.requested_pages to append
```

Comment:
please use english in your comments. This seems to be unfinished docstring.

Reply:
fixed and done